test/JDBC/expected/BABEL-3802-vu-prepare.out

```diff
@@ -41,7 +41,11 @@ CREATE PROCEDURE BABEL_3802_vu_prepare_t_p1 AS (
         power(CAST(NULL AS BIGINT),1),
         power(CAST(8969.32 AS BIGINT),1),
         power(CAST(896932 AS BIGINT),1.1),
-        power(CAST(896932 AS BIGINT),1.2)
+        power(CAST(896932 AS BIGINT),1.2),
+        power(CAST(NULL AS BIGINT),NULL),
+        power(CAST(896932 AS BIGINT),NULL),
+        power(CAST(2 AS BIGINT), -1),
+        power(CAST(0 AS BIGINT),0)
```

Comment:
add the similar tests for view? also, add test cases like `power(2, 2.2)` and `power(2, 2.8)` . overflow, underflow tests are missing.

Reply:
Yes, Added the similar tests for view.